src/Sylius/Component/Promotion/Checker/Eligibility/PromotionUsageLimitEligibilityChecker.php

```diff
@@ -26,11 +26,6 @@ public function isEligible(PromotionSubjectInterface $promotionSubject, Promotio
         if (null === $usageLimit = $promotion->getUsageLimit()) {
             return true;
         }
-
-        if ($promotion->getUsed() < $usageLimit) {
-            return true;
-        }
-
-        return false;
+        return $promotion->getUsed() < $usageLimit;
```

Comment:
In my opinion, the previous version looks easier to read, especially if there are more conditions in the future.

Reply:
The previous solutions is confusing. It suggest there is more than `$promotion->getUsed() < $usageLimit` going on.

In cases like this, it would make sense:

```php
if ($promotion->getUsed() < $usageLimit) {
    return []
}

return null;
```